snakeware/external/configs/x86-64_defconfig

```diff
@@ -39,45 +494,3211 @@ BR2_PACKAGE_ALSA_UTILS_ASEQNET=y
+# BR2_PACKAGE_LIBNSS is not set
+# BR2_PACKAGE_LIBOLM is not set
+# BR2_PACKAGE_LIBP11 is not set
+# BR2_PACKAGE_LIBSCRYPT is not set
+# BR2_PACKAGE_LIBSECRET is not set
+# BR2_PACKAGE_LIBSHA1 is not set
+# BR2_PACKAGE_LIBSODIUM is not set
+# BR2_PACKAGE_LIBSSH is not set
+# BR2_PACKAGE_LIBSSH2 is not set
+# BR2_PACKAGE_LIBTOMCRYPT is not set
+# BR2_PACKAGE_LIBUECC is not set
+# BR2_PACKAGE_MBEDTLS is not set
+# BR2_PACKAGE_NETTLE is not set
+BR2_PACKAGE_OPENSSL_FORCE_LIBOPENSSL=y
+BR2_PACKAGE_OPENSSL=y
+BR2_PACKAGE_LIBOPENSSL=y
+BR2_PACKAGE_LIBOPENSSL_TARGET_ARCH="linux-x86_64"
+# BR2_PACKAGE_LIBOPENSSL_BIN is not set
+# BR2_PACKAGE_LIBOPENSSL_ENGINES is not set
+BR2_PACKAGE_HAS_OPENSSL=y
+BR2_PACKAGE_PROVIDES_OPENSSL="libopenssl"
+BR2_PACKAGE_PROVIDES_HOST_OPENSSL="host-libopenssl"
+# BR2_PACKAGE_RHASH is not set
+# BR2_PACKAGE_TINYDTLS is not set
+# BR2_PACKAGE_TPM2_TSS is not set
+# BR2_PACKAGE_TROUSERS is not set
+# BR2_PACKAGE_USTREAM_SSL is not set
+# BR2_PACKAGE_WOLFSSL is not set
+
+#
+# Database
+#
+# BR2_PACKAGE_BERKELEYDB is not set
+# BR2_PACKAGE_CPPDB is not set
+# BR2_PACKAGE_GDBM is not set
+# BR2_PACKAGE_HIREDIS is not set
+# BR2_PACKAGE_KOMPEXSQLITE is not set
+# BR2_PACKAGE_LEVELDB is not set
+# BR2_PACKAGE_LIBGIT2 is not set
+BR2_PACKAGE_MONGODB_ARCH_SUPPORTS=y
+# BR2_PACKAGE_MYSQL is not set
+# BR2_PACKAGE_POSTGRESQL is not set
+# BR2_PACKAGE_REDIS is not set
+# BR2_PACKAGE_ROCKSDB is not set
+
+#
+# sqlcipher conflicts with sqlite
+#
+BR2_PACKAGE_SQLITE=y
 BR2_PACKAGE_SQLITE_STAT4=y
+# BR2_PACKAGE_SQLITE_ENABLE_COLUMN_METADATA is not set
+# BR2_PACKAGE_SQLITE_ENABLE_FTS3 is not set
+# BR2_PACKAGE_SQLITE_ENABLE_JSON1 is not set
+# BR2_PACKAGE_SQLITE_ENABLE_UNLOCK_NOTIFY is not set
+# BR2_PACKAGE_SQLITE_SECURE_DELETE is not set
+# BR2_PACKAGE_SQLITE_NO_SYNC is not set
+# BR2_PACKAGE_UNIXODBC is not set
+
+#
+# Filesystem
+#
+# BR2_PACKAGE_GAMIN is not set
+# BR2_PACKAGE_LIBCONFIG is not set
+# BR2_PACKAGE_LIBCONFUSE is not set
+# BR2_PACKAGE_LIBFUSE is not set
+# BR2_PACKAGE_LIBLOCKFILE is not set
+# BR2_PACKAGE_LIBNFS is not set
+# BR2_PACKAGE_LIBSYSFS is not set
+# BR2_PACKAGE_LOCKDEV is not set
+# BR2_PACKAGE_PHYSFS is not set
+
+#
+# Graphics
+#
+# BR2_PACKAGE_ASSIMP is not set
+
+#
+# at-spi2-atk depends on X.org
+#
+
+#
+# at-spi2-core depends on X.org
+#
+# BR2_PACKAGE_ATK is not set
+# BR2_PACKAGE_ATKMM is not set
+# BR2_PACKAGE_BULLET is not set
+# BR2_PACKAGE_CAIRO is not set
+# BR2_PACKAGE_CAIROMM is not set
+
+#
+# chipmunk needs an OpenGL backend
+#
+# BR2_PACKAGE_EXEMPI is not set
+# BR2_PACKAGE_EXIV2 is not set
+# BR2_PACKAGE_FONTCONFIG is not set
+# BR2_PACKAGE_FREETYPE is not set
+# BR2_PACKAGE_GD is not set
+# BR2_PACKAGE_GDK_PIXBUF is not set
+# BR2_PACKAGE_GIFLIB is not set
+
+#
+# granite needs libgtk3 and a toolchain w/ wchar, threads
+#
 BR2_PACKAGE_GRAPHITE2=y
+
+#
+# gtkmm3 needs libgtk3 and a toolchain w/ C++, wchar, threads, gcc >= 4.9
+#
 BR2_PACKAGE_HARFBUZZ=y
-BR2_PACKAGE_LIBJPEG=y
```

Comment:
Why did you remove this line?

Reply:
Honestly, I didn't mean to... Some of the initial config editing was through the official buildroot config, and that must have messed a few things up... I only really need to change one, maybe two lines.

Reply:
There are no jpeg files in Snakeware so this probably non-essential...